src/CLICommand/parameters_helper.ts

```diff
@@ -466,7 +465,7 @@ export class ParametersHelper {
 
 		if (!userProvidedURL) {
 			// Return default CLI turbo if no turbo url can be derived from the user
-			return new URL(turboProdUrl);
+			return new URL('https://upload.ardrive.dev');
```

Comment:
TODO: should default to prod. but sending to dev temporarily

```suggestion
			return new URL('https://upload.ardrive.io');
```